Makefile

```diff
@@ -137,9 +137,9 @@ ubuntu:
 
 # Build bins in a golang container, and build the antrea-ubuntu Docker image.
 .PHONY: build-ubuntu
-build-ubuntu:
+build-ubuntu: docker-bin
 	@echo "===> Building Antrea bins and antrea/antrea-ubuntu Docker image <==="
-	docker build -t antrea/antrea-ubuntu -f build/images/Dockerfile.build.ubuntu .
+	docker build -t antrea/antrea-ubuntu -f build/images/Dockerfile.ubuntu .
```

Comment:
There is already a `ubuntu` target doing the same thing, I think the `build-ubuntu` was added to support building image on OSX and windows? Would the change break that?

Reply:
It doesn't because it depends on `docker-bin` which builds the binaries inside the docker.

Reply:
I see, then should we just remove the "build-ubuntu" target and add the "docker-bin" dependency to "build", as it building image always need the binaries, and `go build` can skip compiling when there's no code change.